opm/parser/eclipse/EclipseState/Schedule/Completion.hpp

```diff
@@ -63,11 +65,13 @@ namespace Opm {
         int m_i, m_j, m_k;
         Value<double> m_diameter;
         Value<double> m_connectionTransmissibilityFactor;
+        Value<double> m_wellPi;
```

Comment:
The `Value<T>` datatype is used for situations where an item has a default value, which should be used - e.g.: "If xxx equals -1 - use the alternative algorithm", or no default at all. For the WellPi the default value of 1.0 is perfectly ok to use  - and there is no reason to wrap it in a `Value<double>` - just use plain `double`.